test/utils/hash_test.ts

```diff
@@ -82,12 +82,12 @@ describe("Hash Utility", function () {
 
     context("when user hashed value does not exist", function() {
       it("encrypts the value", async function() {
-        await hashValue(string_param, "test-data-type", "test-jwt");
+        await hashValue("wow", "test-data-type", "test-jwt");
         expect(http.post.calledWith(
           "/user_encrypted_data",
           {
-            value: string_param,
-            hashed_value: expectedHashedValue,
+            value: "wow",
+            hashed_value: "ec52cc2cca29bc83f1d1f6916b885a52ac694d386ea8d1a9798f01aafc75d3bffc94f4d6ef21d354a4992ecbaa784a1d226347a043beac323ce7d500577290f7",
             data_type: "test-data-type"
           },
           "test-jwt"
```

Comment:
can you add a test to verify the hashed_values object now has the new value in it

Reply:
kind of tricky because the hashed values are not exported